app/code/Magento/InventoryGroupedProduct/composer.json

```diff
@@ -4,7 +4,10 @@
   "require": {
     "php": "~7.1.3||~7.2.0",
     "magento/framework": "*",
-    "magento/module-grouped-product": "*"
+    "magento/module-grouped-product": "*",
```

Comment:
Most of the new dependencies seem to be overhead, this module is only supposed to extend Grouped Product.